tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -221,6 +221,26 @@ describe(__filename, () => {
     expect(link).toHaveProp('to', `/addon/${addon.slug}/license/`);
   });
 
+  it('renders a default name when the license name is null', () => {
+    _loadVersions({ license: { name: null, url: 'some-url' } });
+
+    const root = render({});
+
+    const license = root.find('.AddonMoreInfo-license-link');
+    expect(license).toHaveLength(1);
+    expect(license.children()).toIncludeText('Custom License');
+  });
+
+  it('renders a default name when the license name and url are null', () => {
+    _loadVersions({ license: { name: null, url: null } });
+
+    const root = render({});
+
+    const license = root.find('.AddonMoreInfo-license-name');
+    expect(license).toHaveLength(1);
```

Comment:
Maybe also assert that `.AddonMoreInfo-license-link` is not present to make sure it's not an orphaned a tag?